src/ChatWindow.vue

```diff
@@ -152,26 +188,53 @@ export default {
     showDeletion: {
       type: Boolean,
       required: true
+    },
+    myId: {
+      type: String,
+      required: true
     }
   },
   data() {
     return {
-      showUserList: false
+      state: this.initialState()
+    }
+  },
+  watch: {
+    multipleChatsEnabled: function(newMultipleChatsEnabled) {
```

Comment:
could we use a fat arrow func here (just for consistency) 👍

Reply:
I'm changing it to ```
multipleChatsEnabled(newMultipleChatsEnabled) {
      this.state = this.initialState()
    }
```
because this style is mostly used in this file. Ok?